W/Wine/build_tarballs.jl

```diff
@@ -13,9 +13,9 @@ platform_spec = [a for a in ARGS if !startswith(a, "-")]
 is_meta = any(a->startswith(a, "--meta-json"), ARGS)
 requested_platforms = map(p->parse(Platform, p; validate_strict=true), platform_spec)
 
-version = v"7.0-rc1"
+version = v"7.0-rc3"
 sources = Any[
-    GitSource("https://github.com/wine-mirror/wine.git", "533616d23f9832596e41f839356830c7679df930"),
+    GitSource("https://github.com/JuliaComputing/wine-staging.git", "10ff6d27133070cff6369c48d7ee4c56ee8da7aa"),
```

Comment:
It would be nice if we had a kwarg that let us rename the directory. We have `unpack_target`, but that would result in `wine/wine-staging`.

Reply:
>  but that would result in wine/wine-staging

I'm not sure that's how that was supposed to work :confused:

Reply:
Don't know either. I guess that's with analogy to ArchiveSource? Nothing to do here for the moment, but let's track it over in BinaryBuilder and get something fixed for the future.